metals/src/main/scala/scala/meta/internal/metals/FoldingRangeExtractor.scala

```diff
@@ -173,8 +175,9 @@ final class FoldingRangeExtractor(
           startToken <- term.expr.findFirstTrailing(_.is[Token.KwCatch])
           lastCase <- term.catchp.lastOption
           endToken <- lastCase.findFirstTrailing(_.is[Token.RightBrace])
-        } yield
-          Position.Range(tree.pos.input, startToken.pos.end, endToken.pos.end)
+          tryPos = Position
```

Comment:
Formatting here was a bit crazy and worked differently on Windows.

Reply:
What's going on 😨